src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionToolOptions.cs

```diff
@@ -67,15 +71,19 @@ public sealed class RoslynInsertionToolOptions
             bool setAutoComplete,
             ImmutableArray<string> cherryPick)
         {
-            Username = username;
-            Password = password;
+            VisualStudioRepoAzdoUsername = visualStudioRepoAzdoUsername;
```

Comment:
I feel like some of these names could be simplified without much loss of clarity. For example, `visualStudioRepoAzdoUsername` could be `vsUsername` and `componentBuildAzdoUsername` could be `componentUsername`